### DISCUSSION TITLE
请问vc-viewer上clockViewModel如何正确使用？

### DISCUSSION THREAD

**Discussion by Fang-yl** [zouyaoji/vue-cesium#635] [General]
const clock = new Cesium.Clock({
  startTime: Cesium.JulianDate.fromIso8601("2013-12-25"),
  currentTime: Cesium.JulianDate.fromIso8601("2013-12-25"),
  stopTime: Cesium.JulianDate.fromIso8601("2013-12-26"),
  clockRange: Cesium.ClockRange.LOOP_STOP,
  clockStep: Cesium.ClockStep.SYSTEM_CLOCK_MULTIPLIER,
  multiplier: 4000,
  shouldAnimate: true
});

const viewer = new Cesium.Viewer("cesiumContainer", {
  clockViewModel: new Cesium.ClockViewModel(clock),
});
以上是正常使用步骤，求助大神，请问在vc-viewer挂载之后如何使用？感觉文档写的不是很清晰。

**Comment by zouyaoji** [comment]
vc-viewer的ready事件拿到viewer实例后用cesium 的 api 操作。

**Comment by Fang-yl** [comment]
好的，感谢回复。
